aspnetcore/tutorials/razor-pages/index.md

```diff
@@ -23,3 +23,5 @@ This series of tutorials explains the basics of building a Razor Pages web app.
 At the end, you'll have an app that can display and manage a database of movies.
 
 ![Sample page of sample app](index/_static/sample-page.png)
+
+[!INCLUDE[](~/includes/advancedRP.md)]
```

Comment:
You could make this more prominent -
* Insert a line break and a couple blank lines before  "The series includes the following tutorials:" and change it to "This series..."
* Between the blank lines add the include.

Reply:
Can you edit in browser and make those changes?